witchcraft-server/src/metrics/jemalloc.rs

```diff
@@ -19,4 +19,12 @@ pub fn register_metrics(metrics: &MetricRegistry) {
         let _ = epoch::advance();
         stats::allocated::read().unwrap_or(0)
     });
+    metrics.gauge("process.heap.active", move || {
```

Comment:
not entirely sure what to name this/whether process.heap is part of an internal standard naming scheme

Reply:
Yeah I think preserving the jemalloc naming is the right way to go here.